nemo/collections/tts/modules/transformer_2501.py

```diff
@@ -227,11 +227,16 @@ def attn_naive(
 
         # attn_prior or square mask or vanilla attention
         if attn_prior is not None:
-            eps = self.prior_eps
+            eps = 1e-8
```

Comment:
Why do we want to disabled passing in eps?

Reply:
I had added configurable `prior_eps` thinking that setting it to 0 will make the prior strict - but that doesnt work well since it leads to nans. Similarly setting it to a very low value also does not work well, since the learned attention scores learn offset the prior. So I thought it is unnecessary to keep it configurable.